libs/material/test/src/utils/MaterialMother.ts

```diff
@@ -0,0 +1,11 @@
+import { Material, MaterialId, MaterialName, MaterialImage, MaterialIsActive, MaterialDeletedAt} from '@frappe/material/domain';
+import {StringMother, UuidMother } from '@frappe/common/test';
+
+export class MaterialMother {
+  static random(): Material {
+    return new Material(new MaterialId(UuidMother.random()), new MaterialName(StringMother.random()), new MaterialImage(StringMother.random()), new MaterialIsActive(true));
```

Comment:
Esto se podría cambiar para mejorar la legibilidad del código
```suggestion
    return new Material(
      new MaterialId(UuidMother.random()), 
      new MaterialName(StringMother.random()), 
      new MaterialImage(StringMother.random()), 
      new MaterialIsActive(true)
    );
```